README.md

```diff
@@ -139,21 +139,21 @@ class ColorStateViewModel: ViewModel() {
 and to use in Android and iOS you can use ```getColor()``` method which return ```Color``` from Jertpack compose in AndroidMain and ```UIColor``` in iOSMain
 
 ### In Android
-```
+``` kotlin
 Text(text = text, color = getColor(colorState.value))
 ```
 
 ### In iOS
 Since in iOS kotlin has interop with Objective-C and not Swift, we get UIColor which you have to convert to ```Color``` in swift so you can create an extention func in swift
 
-```
+``` swift
 func getColor(kColorRes: KColorRes) -> Color {
     return Color(GetColorKt.getColor(kColorRes: kColorRes))
 }
 ```
 
 and use it like
-```
+``` kotlin
```

Comment:
This one is from swift.

Reply:
Oh. Feel free to edit and force push my commit.